nix/marlowe-cardano/deploy/operables.nix

```diff
@@ -384,7 +384,7 @@ in
       #################
       # REQUIRED VARS #
       #################
-      # HOST, PORT, QUERY_PORT: network binding
+      # HOST, PORT, QUERY_PORT, TRANSFER_PORT: network binding
```

Comment:
Looking for docs regarding this I noticed that `marlowe-protocols` is missing a Readme. Not for this PR of course, but once the protocols settle it would be nice to add some docs around it.